unicon_backend/lib/amqp.py

```diff
@@ -222,16 +264,28 @@ def __init__(
         q_msg_ttl_secs: int | None = None,
         dlx_name: str | None = None,
         dlx_routing_key: str | None = None,
+        reconnect_after_sec: int | None = None,
     ):
         super().__init__(
-            conn_name, amqp_url, ex_name, ex_type, q_name, q_msg_ttl_secs, dlx_name, dlx_routing_key
+            conn_name,
+            amqp_url,
+            ex_name,
+            ex_type,
+            q_name,
+            q_msg_ttl_secs,
+            dlx_name,
+            dlx_routing_key,
+            reconnect_after_sec,
         )
 
         self._deliveries: dict[int, Literal[True]] = {}
         self._acked: int = 0  # Number of messages acknowledged
         self._nacked: int = 0  # Number of messages rejected
         self._message_number: int = 0  # Number of messages published
 
+    def _delivery_callback(self, delivery_tag: int) -> None:
```

Comment:
is this supposed to be an abstract method? if so, probably good to add the abstract method so type checks enforces it

Reply:
Yes! I will add an abstract method for it.